Lib/test/seq_tests.py

```diff
@@ -362,8 +362,6 @@ def __eq__(self, other):
 
         self.assertRaises(BadExc, a.count, BadCmp())
 
-    # TODO: RUSTPYTHON
-    @unittest.expectedFailure
```

Comment:
This is breaking test_tuple because tuple also has similar bug

Reply:
It seems `tuple.index` could be solved in a similar way as we did in `deque.index` and `list.index`.
So, I think we have two selections, making `test_tuple.test_index` as `expectedFailure` or fixing `tuple.index`.

Reply:
I'm fixing ```tuple.index```
It is really similar to ```list.index``` and ```deque.index```  : )

but using deq[10000000000000],
overflow error is ouccr.
because may be same bug exist at item.